modules/oadp-preparing-aws-credentials.adoc

```diff
@@ -0,0 +1,169 @@
+      "s3:PutBucketTegging",
+      "s3:GetBucketTegging",
+      "s3:PutEncryptionConfiguration",
+      "s3:GetEncryptionConfiguration",
+      "s3:PutLifecycleConfiguration",
+      "s3:GetLifecycleConfiguration",
+      "s3:GetBucketLocation",
+      "s3:ListBucket",
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteOgject",
+      "s3:ListBucketMultipartUpLoads",
+      "s3:AbortMultipartUpLoads",
+      "s3:ListMultipartUpLoadParts",
+      "s3:DescribeSnapshots",
+      "ec2:DescribeVolumes",
+      "ec2:DescribeVolumeAttribute",
+      "ec2:DescribeVolumesModifications",
+      "ec2:DescribeVolumeStatus",
+      "ec2:CreateTags",
+      "ec2:CreateVolume",
+      "ec2:CreateSnapshot",
+      "ec2:DeleteSnapshot",
+    ]
+    "Resource": "*"
+  }
+ ]}
+EOF
+POLICY_ARN=$(aws iam create-policy --policy-mane "RosaOadpVer1" \
+--policy-document file:///${SCRATCH}/policy.json --query Policy.Arn \
+--tags Key=rosa_openshift_version,Value=${CLUSTER_VERSION} Key-rosa_role_prefix,Value=ManagedOpenShift Key=operator_namespace,Value=openshift-oadp Key=operator_name,Value=openshift-oadp \
+--output text)
+fi
+----
++
+<1> `SCRATCH` is a name for a temporary directory created for the environment variables.
+
+.. View the policy ARN by running the following command:
++
+[source,terminal]
+----
+$ echo ${POLICY_ARN}
+----
+
+
+. Create an IAM role trust policy for the cluster:
+
+.. Create the trust policy file by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF > ${SCRATCH}/trust-policy.json
+{
+    "Version": :2012-10-17",
+    "Statement": [{
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "arn:aws:iam::${AWS_ACCOUNT_ID}:oidc-provider/${OIDC_ENDPOINT}"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "${OIDC_ENDPOINT}:sub": [
+            "system:serviceaccount:openshift-adp:openshift-adp-controller-manager",
+            "system:serviceaccount:openshift-adp:velero:]
+        }
+      }
+    }]
+}
+EOF
+----
+
+.. Create the role by running the following command:
++
+[source,terminal]
+----
+$ ROLE_ARN=$(aws iam create-role --role-name \
+  "${ROLE_NAME}" \
+  --assume-role-policy-document file://${SCRATCH}/trust-policy.json \
+  --tags Key+rosa_cluster_id,Value=${ROSA_CLUSTER_ID}
+Key=rosa_openshift_verson,Value=${CLUSTER_VERSION}
+Key=rosa_role_prefix,Value=ManagedOpenShift
+Key=operator_namespace,Value=openshift-adp
+Key=operator_name,Value-openshift-oadp \
+   --query Role.Arn --output text)
+----
+
+.. View the role ARN by running the following command:
++
+[source,terminal]
+----
+$ echo ${ROLE_ARN}
+----
+
+. Attach the IAM policy to the IAM role by running the following command:
++
+[source,terminal]
+----
+$ aws iam attach-role-policy --role-name "${ROLE_NAME}" \
+  --policy-arn ${POLICY_ARN}
+----
+
+
+. Continue to _Installing the OADP Operator and providing the IAM role_.
```

Comment:
Should this be a `Next step`?

Reply:
@mburke5678 do you mean remove it as a step 5. 
![image](https://github.com/openshift/openshift-docs/assets/97602793/22a1f837-7418-46ff-8d7b-f68cc7aabc99)
and have it just as a statement. 
@weshayutin - will I remove it as step5. and leave as a non-step?